native/src/modules/layout/containers/HeaderContainer.js

```diff
@@ -32,19 +39,9 @@ type DispatchPropsType = {| dispatch: Dispatch<StoreActionType> |}
 
 type PropsType = {| ...OwnPropsType, ...StatePropsType, ...DispatchPropsType |}
 
-const getShareUrl = (params: {| city: string, language: string, path: ?string, routeName: string |}): ?string => {
-  const { city, language, path, routeName } = params
-  if (path) {
-    return url(path)
-  } else if ([EVENTS_ROUTE, NEWS_ROUTE, POIS_ROUTE].includes(routeName)) {
-    const params = { cityCode: city, languageCode: language, path: null }
-    return cityContentUrl({ route: routeName, ...params })
-  }
-  return null
-}
-
 const mapStateToProps = (state: StateType, ownProps: OwnPropsType): StatePropsType => {
   const routeKey = ownProps.scene.route.key
+  const shareUrlFromScene: ?string = typeof ownProps.scene.route.params?.shareUrl === 'string' ? ownProps.scene.route.params?.shareUrl : null
```

Comment:
I think this is no longer relevant as you removed it completely from the route params, is it?

Reply:
no it is still relevant, e.g. for externalOffers

Reply:
External offers should be external and not have a screen, therefore normally no shareUrl should be necessary :eyes:

Reply:
I meant for example for Lehrstellenradar

Just found: https://www.lehrstellen-radar.de/5100,0,lsrlist.html

Reply:
I see, this thing again :cry: I'd omit the shareUrl here to be honest. Reason 1: The url does not work anyway as it has to be opened with a post request, so if you just klick on it you just get HTTP_METHOD_NOT_SUPPORTED. Reason 2: Even if it would work, the url would be wrong once the user navigates anywhere. Probably we should look into whether it is possible to support sharing the url some other way or just ignore it.

Reply:
But we still need it for `image` and `pdf` routes, nvm. But it is not relevant in the HeaderContainer since those routes are using a transparent header.

Reply:
Should i open a ticket, so we can discuss this in the grooming ?

Reply:
Yeah sure, opening a ticket for sharing the Lehrstellenradar makes sense. I think I'd remove the logic anyway for now as it does not work.